src/bitmap/container.rs

```diff
@@ -68,12 +66,11 @@ impl Container {
         }
     }
 
-    pub fn remove_range(&mut self, start: u32, end: u32) -> u64 {
-        debug_assert!(start <= end);
-        if start == end {
-            return 0;
-        }
```

Comment:
As we are in the `Container` we should accept a `RangeInclusive<u16>` here and return 0 if it `is_empty`.